flink-table/flink-table-planner/src/test/resources/org/apache/flink/table/planner/plan/batch/sql/join/NestedLoopSemiAntiJoinTest.xml

```diff
@@ -673,8 +673,8 @@ Calc(select=[b])
    :     +- HashAggregate(isMerge=[true], groupBy=[EXPR$0], select=[EXPR$0])
    :        +- Exchange(distribution=[hash[EXPR$0]])
    :           +- LocalHashAggregate(groupBy=[EXPR$0], select=[EXPR$0])
-   :              +- Calc(select=[1 AS EXPR$0])(reuse_id=[1])
-   :                 +- LegacyTableSourceScan(table=[[default_catalog, default_database, t, source: [TestTableSource(i, j, k)]]], fields=[i, j, k])
+   :              +- Calc(select=[1 AS EXPR$0])
```

Comment:
This might need investigation. It looks like some quality checks don't work anymore? Otherwise the planner should detect similar nodes.

Reply:
The same reason as mentioned in comment above:
 added optimization in Calcite to remove args for nonDistinct `SqlAggCountFunction` and successors